docs/pages/_document.js

```diff
@@ -9,8 +9,9 @@ import { pathnameToLanguage } from 'docs/src/modules/utils/helpers';
 import { themeColor } from 'docs/src/modules/components/ThemeContext';
 import createCache from '@emotion/cache';
 
-const cache = createCache({ key: 'css', prepend: true });
-const { extractCritical } = createEmotionServer(cache);
+const getCache = () => createCache({ key: 'css', prepend: true });
```

Comment:
This shouldn't break anything (I think), but the overall recommendation is to use custom keys when creating caches and the `"css"` is kinda a default one.

Reply:
Should this be identical with the one we use on the client? For example use `mui` on both places?

Reply:
Yes, definitely it should be the same - this is used for rehydrating stuff and so on.